sql-cli/Makefile

```diff
@@ -31,7 +31,7 @@ pre-commit:  # Run pre-commit
 flow:  # Build and run the latest version of flow
 
 test:  # Run tests
-	@_TYPER_FORCE_DISABLE_TERMINAL=1 PYTHONWARNINGS="ignore" poetry run pytest -s --cov --cov-branch --cov-report=term-missing tests
+	@_TYPER_FORCE_DISABLE_TERMINAL=1 PYTHONWARNINGS="ignore" poetry run pytest
```

Comment:
@pankajkoti why did we remove the coverage stats?

Reply:
@feluelle can we add them back or you have a different opinion?

Reply:
It is now in the project.toml

Reply:
I guess Felix moved them to pyproject.toml here: https://github.com/astronomer/astro-sdk/pull/1258/files#diff-f92d64397e38a0a17a38a3b24f90b96020f886a2f8db484165658088325e0c9bR88-R93

Reply:
This way we have the options in all pytest runs - local and in CI.